docs/faq.md

```diff
@@ -29,6 +32,12 @@
 
     Job is a collection of tasks (a task graph). You can display and manage jobs using the CLI.
 
+??? question "What operating systems does HQ support?"
+
+    HyperQueue currently only officially supports Linux (Ubuntu, Debian, CentOS, etc.). It might be possible to
```

Comment:
I would not named distributions as it seems that we need some support for specific each of Linux distributions.

Reply:
This si copy-pasted from README, I didn't edit it, btw xD Removed.